src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -100,4 +100,9 @@ public Conversation getConversationWithTitle(String title) {
   public void setConversations(List<Conversation> conversations) {
     this.conversations = conversations;
   }
+
+  /** Gets the total number of Conversations stored by this ConversationStore. */
+  public int getTotalConversations() {
```

Comment:
I think adding these size methods to each of the Store classes is going to complicate your code unnecessarily.  I think you'd be better off reusing the `getAllConversations` method and just doing `getAllConversations().size()`.

Reply:
Fixed!